h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -274,6 +269,42 @@ protected void setupLocal() {
       }
     }
 
+    /**
+     * @return Number of connections to an SQL database to be opened on a single node.
+     */
+    int getMaxConnectionsPerNode(final int cloudSize, final short nThreads) {
+      int conPerNode;
+      final String userDefinedMaxConnections = System.getProperty(MAX_USR_CONNECTIONS_KEY);
+        try {
+          Integer connectionAmount = Integer.valueOf(userDefinedMaxConnections);
+          if (connectionAmount > 0 && connectionAmount < MAX_CONNECTIONS) {
+            conPerNode = calculateLocalConnectionCount(connectionAmount, cloudSize, nThreads);
+          } else {
+            conPerNode = calculateLocalConnectionCount(MAX_CONNECTIONS, cloudSize, nThreads);
```

Comment:
Why do we not respect users setting? I would respect what the user wants and issue a warning